src/config.rs

```diff
@@ -395,10 +393,7 @@ impl Config {
 
     /// Whether the config option needs an IO scheduler restart to take effect.
     pub(crate) fn needs_io_restart(&self) -> bool {
-        matches!(
-            self,
-            Config::MvboxMove | Config::OnlyFetchMvbox | Config::SentboxWatch
```

Comment:
Removed `MvboxMove` because IO restart is needed now only for the chatmail case where the mvbox isn't created by default. But for chatmail we don't expose this setting to users anyway